### DISCUSSION TITLE
Can Huawei smartwatch GT 2-319 be used as a standalone collector?

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2286] [Q&A]
@AliSaeed86
I have copied your question to here, discussions.  
My hope is that these discussions become the hub for all the questions.

I am also copying Patrick, who may be able to help you much much much better than I can with respect to watches.
@psonnera

Hi @Garweho, @Navid200,
Kindly, can Huawei smartwatch GT 2-319 be used as a standalone collector?
I tried too many times to set my Huawei SW GT 2 to collect Libre1 readings, but no use. There is no play store in the smartwatch to install xdripe+.

Please, help me.
Note I am using MaioMaio1, Libre1, xdrip+ and Samsung mobile.

Thanks.

**Comment by AliSaeed86** [comment]
Thanks for redirecting me 💐🌹

**Reply by psonnera** [reply]
GT2 is not Android Wear, so no.
I don't know of anybody developing anything with this OS.

**Reply by AliSaeed86** [reply]
How could i know the android watch,  pls? Any specific brand names pls?

Thanks

**Reply by psonnera** [reply]
Theoretically any watch running Android Wear. https://wearos.google.com/#find-your-watch
